MarsOnboardingTask/Utils/Driver.cs

```diff
@@ -0,0 +1,35 @@
+﻿
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+namespace MarsOnboardingTask.Utils;
+
+
+public class Driver
+{
+    public static IWebDriver driver = new ChromeDriver();
```

Comment:
you should intialise this object inside the method below like like so
    public void Initialize()
    {
driver = new ChromeDriver();
        Wait(driver, 10);
        driver.Manage().Window.Maximize();
    }

Reply:
updated as per the review comments